src/components/Create/Create.jsx

```diff
@@ -134,12 +231,54 @@ function Create(props) {
         )
     }
 
+    // data table with sites
+    const renderTable = () => {
+        return (
+            <>
+            <DataTable scrollHeight="350px">
+                <TableHead>
+                    <DataTableRow>
+                        {errorData && errorData.fields.map((field, index) => {
+                             return (
+                             <DataTableColumnHeader key={index} fixed top="0">
+                                 {field}
+                             </DataTableColumnHeader>
+                             )
+                        })}
+                    </DataTableRow>
+                </TableHead>
+                <TableBody>
+                    {errorData && errorData.errors.map((rowData, index) => {
+                        return (
+                            <DataTableRow key={`row-${index}`}>
+                                {Object.values(rowData).map((error, index) => {
+                                    return <DataTableCell key={`cell-${index}`}>{error}</DataTableCell>
+                                })}
+                                
+                            </DataTableRow>
+                            )
+                    })}
+                </TableBody>
+            </DataTable>
+            </>
+        )
+    }
     return <Modal>
         <ModalTitle>{title}</ModalTitle>
         <ModalContent>
             {renderForm()}
+            <Divider/>
+            {hasErrors && <Modal>
+                <ModalTitle>{errorData.errors.length} {i18n.t("errors were found")}</ModalTitle>
+                <ModalContent>
+                    {i18n.t("Click proceed if you want to continue and the errors will automatically be removed or click cancel to go back.")}
```

Comment:
I think we should clarify that the sites with errors will be removed (not just the errors themselves)